omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -2823,36 +2825,47 @@
                                                     <repeat>
                                                         <template>
                                                             <obsgroup groupingConceptId="160741">
-                                                                <div class="row multi-drug" id="{id}-toggleContainer">
-                                                                    <div class="col-2">
-                                                                        <obs labelText="Drugs" id="{id}-drug"
-                                                                             conceptId="1282" answerClasses="Drug"
-                                                                             style="autocomplete" class="fullwidth"/>
-                                                                    </div>
-                                                                    <div class="col-2">
-                                                                        <obs labelText="Quantity" conceptId="160856"/>
-                                                                    </div>
-                                                                    <div class="col-2">
-                                                                        <obs labelText="Qty unit" conceptId="165791"
-                                                                             answerConceptSetIds="162402,162384"/>
-                                                                    </div>
-                                                                    <div class="col-2 duration">
-                                                                        <label>Duration</label>
-                                                                        <obs conceptId="159368"/>
-                                                                    </div>
-                                                                    <div class="col-2">
-                                                                        <obs conceptId="1732"
-                                                                             labelText="Duration unit"/>
+                                                                <div class="div-row multi-drug" id="{id}-toggleContainer">
```

Comment:
Why are you changing this section. Its not called for

Reply:
@slubwama  I introduced 3 additional fields i which could not fit in the same row, thus overlapping.